meteor/client/ui/Settings/RundownLayoutEditor.tsx

```diff
@@ -1576,7 +550,7 @@ export default translateWithTracker<IProps, IState, ITrackedProps>((props: IProp
 
 			return (
 				<div className="studio-edit rundown-layout-editor">
-					<h2 className="mhn">{t('Shelf Layouts')}</h2>
+					<h2 className="mhn">{this.props.customRegion.title}</h2>
```

Comment:
This used to be translated, now will always be in Code-English. I think this could be mitigated by making `GetSettingsManifest()` translation-aware.

Reply:
I've added the requirement to pass a `TFunction` to `getSettingsManifest`, if that will do the job? https://github.com/olzzon/tv-automation-server-core/commit/b94a303a47a7db47a349a4f715822b3496341afe